tutorials/terraform/data-migration-mysql-mmy/data-migration-mysql-mmy.tf

```diff
@@ -0,0 +1,134 @@
+# Infrastructure for Yandex Cloud Managed Service for MySQL cluster and Virtual Machine.
+#
+# RU: https://cloud.yandex.ru/docs/managed-mysql/tutorials/data-migration
+# EN: https://cloud.yandex.com/en/docs/managed-mysql/tutorials/data-migration
+#
+# Set the following settings:
+locals {
+  # Managed Service for MySQL cluster.
+  target_mysql_version = "" # Set the MySQL version. It must be the same or higher than the version in the source cluster.
+  target_sql_mode      = "" # Set the MySQL SQL mode. It must be the same as in the source cluster.
+  target_db_name       = "" # Set the target cluster database name.
+  target_user          = "" # Set the target cluster username.
+  target_password      = "" # Set the target cluster password.
+  # (Optional) Virtual Machine.
+  vm_image_id   = "" # Set a public image ID from https://cloud.yandex.com/en/docs/compute/operations/images-with-pre-installed-software/get-list.
+  vm_username   = "" # Set a username for VM. Images with Ubuntu Linux use username `ubuntu` by default.
```

Comment:
Set a username for VM. Images with Ubuntu Linux use the username `ubuntu` by default.